pxr/imaging/plugin/hdRpr/sphereLight.cpp

```diff
@@ -53,14 +53,17 @@ RprApiObject HdRprSphereLight::CreateLightMesh(std::map<TfToken, float> & params
 
 GfVec3f HdRprSphereLight::NormalizeLightColor(const GfMatrix4d & transform, std::map<TfToken, float> & params, const GfVec3f & inColor)
 {
+	// Approximating the area of a stretched ellipsoid using the Knud Thomsen formula:
+	// http://www.numericana.com/answer/ellipsoid.htm
 	constexpr const double p = 1.6075;
 	constexpr const double pinv = 1. / 1.6075;
+	const float radius = params[HdRprSphereLightTokens->radius];
 
-	const double sx = GfVec3d(transform[0][0], transform[1][0], transform[2][0]).GetLength();
-	const double sy = GfVec3d(transform[0][1], transform[1][1], transform[2][1]).GetLength();
-	const double sz = GfVec3d(transform[0][2], transform[1][2], transform[2][2]).GetLength();
+	const double sx = GfVec3d(transform[0][0], transform[1][0], transform[2][0]).GetLength() * radius;
+	const double sy = GfVec3d(transform[0][1], transform[1][1], transform[2][1]).GetLength() * radius;
+	const double sz = GfVec3d(transform[0][2], transform[1][2], transform[2][2]).GetLength() * radius;
 
```

Comment:
BTW we could also add an optimization here of course if SX == SY == SZ not a huge deal though

Reply:
Hmm - yeah, the approximation is exact when SX == SY == SZ, so we could switch over to the optimization without worrying about any "jump"...